src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -197,7 +190,14 @@ class AssemblerConnectionManager(
         dataSource.connection.use { connection ->
             connection.createStatement().use { statement ->
                 statement.execute(
-                        "ALTER ROLE ${assemblerConfiguration.server["username"]} SET search_path to $INTEGRATIONS_SCHEMA,$MATERIALIZED_VIEWS_SCHEMA,$PUBLIC_SCHEMA"
+                        setSearchPathSql(
+                                assemblerConfiguration.server["username"].toString(),
+                                false,
+                                INTEGRATIONS_SCHEMA,
+                                MATERIALIZED_VIEWS_SCHEMA,
+                                PUBLIC_SCHEMA,
```

Comment:
Pretty sure we delete this schema, but I guess it can't hurt.

Reply:
the public schema? Yeah I have no idea what it's for but we seem to do quite a bit of work around revoking/maintaining some set of permissions on it, so it seems like it was important at least at some point in time